pkg/apis/pravega/v1alpha1/pravegacluster_types.go

```diff
@@ -19,12 +19,15 @@ const (
 	// DefaultZookeeperUri is the default ZooKeeper URI in the form of "hostname:port"
 	DefaultZookeeperUri = "zk-client:2181"
 
+	// DefaultBookkeeperUri is the default ZooKeeper URI in the form of "hostname:port"
```

Comment:
Just wondering if we can have a Uri like `DefaultZookeeperUri = "zk-client:2181"`? If we have like 6 Bookies then user need to manually type a bunch of words. 
Maybe we can have something like 
```
Bookkeeper:
  name: 
  replicas:
```
and we generate the name for users. What do you think?

Reply:
Good point. However, we can't **assume** that the BK deployment used with Pravega will be a statefulset with a headless service and that users will use the headless service URLs for Bookkeeper here. One option could be to create a client service for the  Bookkeeper Cluster similar to ZK and expect them to provide the URL for the same.

Reply:
Headless services can also be referenced using the service URL instead for hostnames, like: pravega-bk-headless.default.svc.cluster.local
See: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services